src/ensembl/src/content/app/browser/browser-cog/BrowserCog.tsx

```diff
@@ -29,13 +31,21 @@ const BrowserCog = (props: BrowserCogProps) => {
     }
   }, [cogActivated]);
 
-  const inline: CSSProperties = { position: 'relative' };
   const imgInline: CSSProperties = {
     height: '18px',
     width: '18px'
   };
 
-  const cogIcon = props.cogActivated ? cogOnIcon : cogOffIcon;
+  const cogIconConfig = {
+    description: 'Track search',
+    icon: cogIcon,
+    name: 'search'
+  };
+
+  const getCogIconStatus = () => {
+    const { cogActivated } = props;
+    return cogActivated ? Status.HIGHLIGHTED : Status.ACTIVE;
```

Comment:
I don't think the opposite status of `ACTIVE` should be `HIGHLIGHTED`. `INACTIVE` would be more suitable for this.

Reply:
As I said earlier we need to discuss this functionality.

Reply:
I just had a look at this button again. Seems like `DEFAULT` is better suited than `INACTIVE`.